src/lib/project.js

```diff
@@ -142,6 +142,12 @@ async function project({ name, ui }) {
   // `/dev/null` on Mac or Linux and 'NUL' on Windows is the only way to silence
   // Husky's install log msg. (Note: The contract project template commits
   // package-lock.json so we can use `npm ci` for faster installation.)
+
+  // Add a ts-watch script to the contract package.json
+  let contractPkgJson = fs.readJSONSync('package.json');
+  contractPkgJson.scripts['buildw'] = 'tsc -p tsconfig.json --watch';
+  fs.writeJSONSync('package.json', contractPkgJson, { spaces: 2 });
```

Comment:
any reason why we don't put this script in the template `package.json` directly?

Reply:
Good call. I removed this and added it directly to the template.